_data/IshanPradhan/IshanPradhan.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Ishan Pradhan
+institution: Shri Vaishnav Vidyapeeth Vishwavidyalaya
+profile_pic: Ishan.jpg 
```

Comment:
Incorrect file name, please do one of the following:
1. Change `Ishan.jpg ` to `Ishan.jpeg ` in the markdown file.
2. Rename `Ishan.jpeg ` to `Ishan.jpg`.
I'd suggest you make the change in the markdown file!

Reply:
Done! 
Do i need o make another Pull Request?

Reply:
Not needed, your pull request is perfect! :confetti_ball: I approved the changes. :+1:

Reply:
Thanks! :D